src/code/z_lifemeter.c

```diff
@@ -1,59 +1,147 @@
 #include "global.h"
 
-extern s16 D_8011FF10[3];
-extern s16 D_8011FF24[3];
-extern s16 D_8011FF38[3];
-extern s16 D_8011FF4C[3];
-extern s16 D_8011FF60[3];
-extern s16 D_8011FF74[3];
-extern s16 D_8011FF88[3];
-extern s16 D_8011FF9C[3];
-extern u8* D_8011FFB0[];
-extern u8* D_8011FFF0[];
-
-s16 D_8015FDC0[3];
-s16 D_8015FDC8[3];
-s16 D_8015FDD0[6];
-s16 D_8015FDE0[6];
-
-void Health_InitData(GlobalContext* globalCtx) {
+/***
+ * These are the colors for the hearts in the interface. The prim color is the red color of the heart
+ * for the base hearts, while the prim color for the double defense hearts is the white outline. The
+ * env color for the base hearts is the purple-ish outline, while the env color for the double defense
+ * hearts is the red color of the hearts.
+ ***/
+
+#define HEART_PRIM_R 255
+#define HEART_PRIM_G 70
+#define HEART_PRIM_B 50
+
+#define HEART_ENV_R 50
+#define HEART_ENV_G 40
+#define HEART_ENV_B 60
+
+#define HEART_DOUBLE_DEFENSE_PRIM_R 255
+#define HEART_DOUBLE_DEFENSE_PRIM_G 255
+#define HEART_DOUBLE_DEFENSE_PRIM_B 255
+
+#define HEART_DOUBLE_DEFENSE_ENV_R 200
+#define HEART_DOUBLE_DEFENSE_ENV_G 0
+#define HEART_DOUBLE_DEFENSE_ENV_B 0
+
+/***
+ * The burn and drown colors listed here are unused. Prerelease footage of the game confirms that at one
+ * point in development the orange color was to be used while taking damage from hot environments.
+ * Based on this, we can assume that the blue heart color was to be used while drowning.
+ * In the final game these environments only have a timer and do not damage you continuously.
+ ***/
+
+#define HEART_BURN_PRIM_R 255
+#define HEART_BURN_PRIM_G 190
+#define HEART_BURN_PRIM_B 0
+
+#define HEART_PRIM_BURN_ENV_R 255
+#define HEART_PRIM_BURN_ENV_G 0
+#define HEART_PRIM_BURN_ENV_B 0
```

Comment:
I think this is what you meant?
```suggestion
#define HEART_BURN_ENV_R 255
#define HEART_BURN_ENV_G 0
#define HEART_BURN_ENV_B 0
```